### PR TITLE
Splits languages into languages_spoken and languages_understood.

### PR DESCRIPTION
**Pull request by Cruix**
In order to get rid of the hacky drone say code.

### CODE REVIEW

File: code/game/gamemodes/handofgod/god.dm [hunk 1]
```diff
@@ -8,7 +8,8 @@
 	see_in_dark = 0
 	see_invisible = 55
 	sight = SEE_TURFS | SEE_MOBS | SEE_OBJS | SEE_SELF
-	languages = HUMAN | MONKEY | ALIEN | ROBOT | SLIME | DRONE | SWARMER
+	languages_spoken = HUMAN | MONKEY | ALIEN | ROBOT | SLIME | DRONE | SWARMER
```
Comment (RemieRichards) [inline code/game/gamemodes/handofgod/god.dm]: these should use ```ALL```

**Comment by Cruix** [comment]
Well that was fast, I pulled not 30 minutes ago

File: code/game/gamemodes/handofgod/god.dm (hunk 1 above)
Comment (Cruix) [inline code/game/gamemodes/handofgod/god.dm]: Is there a reason to have an ALL_LANGS that is different from ALL?

Comment (RemieRichards) [inline code/game/gamemodes/handofgod/god.dm]: well ```ALL``` sounds like it's gonna get used for ANYTHING, which worries me people will assume it's unused, and thus define their own ```ALL```'s, thus breaking all the ... ```ALL```s.

Comment (Cruix) [inline code/game/gamemodes/handofgod/god.dm]: ALL is just a generic define in flags.dm with all the bits set to 1

Comment (RemieRichards) [inline code/game/gamemodes/handofgod/god.dm]: I see, guess it's immune then.

File: code/game/gamemodes/miniantags/revenant/revenant.dm [hunk 2]
```diff
@@ -0,0 +1,443 @@
+//Revenants: based off of wraiths from Goon
+//"Ghosts" that are invisible and move like ghosts, cannot take damage while invisible
+//Don't hear deadchat and are NOT normal ghosts
+//Admin-spawn or random event
+
+#define INVISIBILITY_REVENANT 50
+
+/mob/living/simple_animal/revenant
```
Comment (X-TheDark) [inline code/game/gamemodes/miniantags/revenant/revenant.dm]: LONG LIVE REVENANT

Comment (Cruix) [inline code/game/gamemodes/miniantags/revenant/revenant.dm]: wat I deleted that

File: code/game/gamemodes/handofgod/god.dm (hunk 1 above)
Comment (MrStonedOne) [inline code/game/gamemodes/handofgod/god.dm]: The question is rather or not you want it to apply to ALL future languages too

**Comment by optimumtact** [comment]
I'm gonna give this another 24 hours but then I'm gonna merge unless any other maintainers spot problems

@tgstation/commit-access

**Pull request merged**